src/wallet/feebumper.h

```diff
@@ -23,39 +23,34 @@ enum class BumpFeeResult
     MISC_ERROR,
 };
 
-class CFeeBumper
-{
-public:
-    CFeeBumper(const CWallet *pWalletIn, const uint256 txidIn, const CCoinControl& coin_control, CAmount totalFee);
-    BumpFeeResult getResult() const { return currentResult; }
-    const std::vector<std::string>& getErrors() const { return vErrors; }
-    CAmount getOldFee() const { return nOldFee; }
-    CAmount getNewFee() const { return nNewFee; }
-    uint256 getBumpedTxId() const { return bumpedTxid; }
-
-    /* signs the new transaction,
-     * returns false if the tx couldn't be found or if it was
-     * impossible to create the signature(s)
-     */
-    bool signTransaction(CWallet *pWallet);
-
-    /* commits the fee bump,
-     * returns true, in case of CWallet::CommitTransaction was successful
-     * but, eventually sets vErrors if the tx could not be added to the mempool (will try later)
-     * or if the old transaction could not be marked as replaced
-     */
-    bool commit(CWallet *pWalletNonConst);
-
-private:
-    bool preconditionChecks(const CWallet *pWallet, const CWalletTx& wtx);
-
-    const uint256 txid;
-    uint256 bumpedTxid;
-    CMutableTransaction mtx;
-    std::vector<std::string> vErrors;
-    BumpFeeResult currentResult;
-    CAmount nOldFee;
-    CAmount nNewFee;
-};
+/* return whether transaction can be bumped */
+bool TransactionCanBeBumped(CWallet* pWallet, const uint256& txid);
+
+/* create bumpfee transaction */
+BumpFeeResult CreateBumpFeeTransaction(const CWallet* pWallet,
```

Comment:
One line?

Reply:
> One line?

I use clang-format for everything, and avoid manual formatting, so that's where this comes from. Would prefer not to change unless someone thinks this is a problem.